demos/src/demo-color.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const d3Selection = require('d3-selection');
+const colors = require('./../../src/charts/helpers/color');
+
+function createColors() {
+    const { colorSchemas } = colors;
+
+    const colorPalettes = Object.keys(colorSchemas).map(colorPalette => {
+        const paletteDiv = document.getElementById(`js-palette-${colorPalette}`);
+
+        paletteDiv.style.display = 'flex';
+
+        colorSchemas[colorPalette].map(color => {
+            const div = document.createElement('div');
+
+            div.style.width = '100px';
+            div.style.height = '100px';
+            div.style.background = color;
+            paletteDiv.appendChild(div);
+        });
+    });
+
+    return colorPalettes;
```

Comment:
If you use `forEach`, you can remove `colorPalettes` variable and not return anything.